### PR TITLE
allow for svg output

### PR DESCRIPTION
**Pull request by mbarkhau**

### CODE REVIEW

**Comment by ecarreras** [comment]
Thank you! :cake: The test fails but is due a `blockdiag` package, nothing to worry about your commit.

File: markdown_blockdiag/extension.py [hunk 1]
```diff
@@ -6,7 +6,8 @@ class BlockdiagExtension(Extension):
 
     def __init__(self, **kwargs):
         self.config = {
-            'fontpath': ['', 'Font path to use']
+            'fontpath': ['', 'Font path to use'],
+            'format': ['', 'Format to use (png/svg)'],
```
Comment (ecarreras) [inline markdown_blockdiag/extension.py]: to be compatibility backward put here `png`

File: markdown_blockdiag/parser.py [hunk 2]
```diff
@@ -19,13 +19,17 @@ def test(self, parent, block):
         return bool(self.RE.match(block))
 
     def run(self, parent, blocks):
-
         raw_block = blocks.pop(0)
         font_path = self.extension.getConfig('fontpath')
-        diagram = draw_blockdiag(raw_block, font_path=font_path)
+        output_fmt = self.extension.getConfig('format', 'png')
```
Comment (ecarreras) [inline markdown_blockdiag/parser.py]: Putting as default value in the definition you don't need to put here.

**Review by ecarreras — CHANGES_REQUESTED** [review]
With these changes the tests pass

File: markdown_blockdiag/parser.py (hunk 2 above)
Comment (mbarkhau) [reply markdown_blockdiag/parser.py]: You're correct. Also I'm working on the tests.

**Comment by mbarkhau** [comment]
Yes, the tests pass, but the build doesn't, because of coverage.

**Comment by mbarkhau** [comment]
I think the build had a hiccup.

**Comment by mbarkhau** [comment]
https://travis-ci.org/gisce/markdown-blockdiag/jobs/208160182 <- the coverage build was ok afaict.

**Review by mbarkhau — COMMENTED** [review]
The default needed to be added here for existing code not to break.

**Comment by mbarkhau** [comment]
Sorry, I don't know how to deal with this "Changes requested". I think I have everything taken care of as far as your suggestions are concerned and the coverage shouldn't be an issue either.

**Comment by mbarkhau** [comment]
This is too messy, starting a new one.

**Pull request closed without merging**